.github/workflows/oxfmt-ci.yml

```
@@ -0,0 +1,214 @@
+name: oxfmt ecosystem ci
+
+on:
+  schedule:
+    - cron: "0 */6 * * *" # Every 6 hours
+  pull_request:
+    types: [opened, synchronize]
+    paths:
+      - "oxfmt-matrix.json"
+      - ".github/workflows/oxfmt-ci.yml"
+      - "bin/clone-oxfmt.js"
+      - "bin/test-oxfmt.js"
+  push:
+    branches:
+      - main
+    paths:
+      - "oxfmt-matrix.json"
+      - ".github/workflows/oxfmt-ci.yml"
+      - "bin/clone-oxfmt.js"
+      - "bin/test-oxfmt.js"
+  workflow_dispatch:
+    inputs:
+      issue-number:
+        description: "Issue number for comment"
+        required: false
+        type: string
+      comment-id:
+        description: "Comment ID to update"
+        required: false
+        type: string
+      ref:
+        description: "Git ref to build from"
+        required: false
+        type: string
+        default: "main"
+
+jobs:
+  reply:
+    name: Reply
+    if: ${{ inputs.issue-number && inputs.comment-id }}
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      contents: write
+    steps:
+      - uses: peter-evans/create-or-update-comment@e8674b075228eee787fea43ef493e45ece1004c9 # v5
+        with:
+          token: ${{ secrets.OXC_BOT_PAT }}
+          repository: oxc-project/oxc
+          issue-number: ${{ inputs.issue-number }}
+          comment-id: ${{ inputs.comment-id }}
+          reactions: eyes
+
+  build:
+    name: Build oxfmt
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+        with:
+          repository: oxc-project/oxc
+          ref: ${{ inputs.ref }}
+          persist-credentials: false
+          show-progress: false
+
+      - uses: oxc-project/setup-rust@1ff88fdaffd6ae35c0fb32ccc159340b37d0beac # v1.0.7
+        with:
+          save-cache: ${{ github.ref_name == 'main' }}
+
+      - uses: oxc-project/setup-node@fdbf0dfd334c4e6d56ceeb77d91c76339c2a0885 # v1.0.4
+
+      - uses: pnpm/action-setup@41ff72655975bd51cab0327fa583b6e92b6d3061 # v4
+
+      - run: pnpm install
+
+      - run: pnpm --filter ./apps/oxfmt run build --detect_code_removal
+        env:
+          RUSTFLAGS: "-C debug-assertions=true"
```

```suggestion
      # Equivalent to `pnpm build` with `detect_code_removal` feature
      - run: pnpm --filter ./apps/oxfmt run build-napi --release --features allocator,detect_code_removal
        env:
          RUSTFLAGS: "-C debug-assertions=true"
      - run: pnpm --filter ./apps/oxfmt run build-js
```

Currently, this CI is the only flag consumer for `oxfmt`, so I think this should be fine for now.

What do you think?